src/BuildingRegistry/Building/Building_BuildingUnit.cs

```diff
@@ -239,6 +239,15 @@ public void CorrectDeregulationBuildingUnit(BuildingUnitPersistentLocalId buildi
                 .DetachAddressBecauseAddressWasRemoved(addressPersistentLocalId);
         }
 
+        public void ReplaceAddressAttachmentFromBuildingUnitBecauseStreetNameWasReaddressed(
+            BuildingUnitPersistentLocalId buildingUnitPersistentLocalId,
+            AddressPersistentLocalId sourceAddressPersistentLocalId,
```

Comment:
Can we also rename sourceAddressPersistentLocalId and destinationAddressPersistentLocalId here?